UC-Berkeley-Extension/global
# Column Section component

## Asana Ticket(s)
- https://app.asana.com/0/0/1197462532271955/f
- https://app.asana.com/0/0/1197462532271957/f

## Description
- Created Column Section component in styleguide & Drupal
- Created Video Embed & Body Copy sub-components in styleguide & Drupal
- Created paragraph templates and hooked up the components
- Created media type Video to go with Video Embed

## Testing instructions
- Compile the styleguide: move to bglobal theme root, run `npm start`
- Back in project root, `ddev start` then ssh into the container
- `drush cim`, `drush cr` (or use the `refresh.sh` script in the README)
- Create a Landing Page, on the main Berkeley select option
- Verify that you have options of creating Panorama, Column Section, and Info Row
- Create a Column Section. Add between 1-3 sub-components, save.
- Verify that it goes full width, and it looks as it should.
<img width="500" alt="Screen Shot 2020-10-07 at 9 21 24 AM" src="https://user-images.githubusercontent.com/10488517/95343747-7e222300-087e-11eb-885d-9fd104a91d0c.png">
